mod_modular_vanilla/hooks/skills/actives/reload_handgonne_skill.nut

```diff
@@ -0,0 +1,11 @@
+::ModularVanilla.MH.hook("scripts/skills/actives/reload_handgonne_skill", function(q) {
+	// Convert the vanilla method of "setting" certain fields to instead be incremental changes
+	q.onAfterUpdate = @() function( _properties )
+	{
+		if (_properties.IsSpecializedInCrossbows)
+		{
+			this.m.FatigueCostMult *= ::Const.Combat.WeaponSpecFatigueMult;
+			this.m.ActionPointCost -= 3;
```

Comment:
This should make sure the cost does not go into the negatives. We might even wanna cap it to 1.

Reply:
Thanks, will fix.